packages/web/src/components/gcds-search/gcds-search.tsx

```diff
@@ -78,6 +83,13 @@ export class GcdsSearch {
    */
   @State() lang: string;
 
+  private onInput = e => {
+    const input = e.target as HTMLInputElement;
+    this.value = input.value;
+
+    this.gcdsChange.emit(this.value);
+  };
+
```

Comment:
I have a comment in this PR that also applies to this -- same situation about thinking about having a `gcdsInput` for consistency